configure.in

```diff
@@ -9,7 +9,9 @@ dnl
 dnl
 dnl
 AC_PREREQ(2.13)
-AC_INIT(src/main.c)
+AC_CONFIG_SRCDIR([src/main.c])
+AC_INIT([ctxusb_daemon], [1.0.1])
```

Comment:
Is there a reason to use the name of the old daemon here? Shouldn't this be "vusb_daemon"?

Reply:
None.
This was in the patch from the layer, will change.